jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1022,7 +1022,14 @@ ecma_op_typedarray_get_to_string_at_index (ecma_object_t *obj_p, /**< this objec
   }
   else
   {
-    ret_value = ecma_op_to_string (index_value);
+    ecma_string_t *str_p = ecma_op_to_string (index_value);
+    if (JERRY_UNLIKELY (str_p == NULL))
+    {
+      ecma_free_value (index_value);
```

Comment:
This free could be merged with the one on line 1035 if done right after the to_string call.